vsts/scripts/tagCliImagesForRelease.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
nit: we need to create a PR in MCR repo for these new images

Reply:
Done ☺️